biolink-model.yaml

```diff
@@ -7104,6 +7110,7 @@ classes:
       - KEGG.DISEASE ## H number
       - HP
       - MP
+      - PHARMGKB:DISEASE
```

Comment:
@andrewsu - I think the tests are complaining that this should be `PHARMGKB.DISEASE` (dot instead of colon). 
I'm happy to give you branch permission here if you'd like to work directly on the model instead of in a fork?  (I could fix for you here, but I would have to branch your fork and it might be easier for you to make the change -- I'll add you here for next time).